data-raw/R/create_fishery_index.R

```diff
@@ -3,14 +3,28 @@
 library(here); library(survdat); library(data.table); library(sf)
 
 #Load landings data
-load(here::here('data-raw', 'data', 'comland.RData'))
+load(here::here('data-raw', 'data', 'landings.RData'))
+
+#Load discards data
+load(here::here('data-raw', 'data', 'discards.RData'))
 
 #Sum over all catch variables for total catch
 spp <- as.numeric(unique(mskeyrun::focalSpecies$NESPP3))
-catchIndex <- comland$comland[NESPP3 %in% spp, .(value = sum(SPPLIVMT)),
+
+#Landings
+landIndex <- landings$comland[NESPP3 %in% spp, .(value = sum(SPPLIVMT)),
                               by = c('NESPP3', 'YEAR')]
-catchIndex[, variable := 'commerical landings']
-catchIndex[, units    := 'metric tons']
+landIndex[, variable := 'commerical landings']
```

Comment:
@slucey I just noticed that `commercial` has a typo here
(I didn't even see it but my code broke--I wasn't calling the variable before so didn't catch it)

Reply:
I changed it in this file but dont have the source data to correct the dataset

Reply:
Thanks for "catching" that...I fixed the code and updated the data object.